### PR TITLE
Change the posts view payload

### LINKED ISSUE #41: Change the posts view payload
The GET posts payload is returning more data than needed, what could be a bottle neck in the future as the database increases. 

Currently, once we call the `/posts` it returns all the database posts with the a HUGE payload, with all the post information and all the related objects with it, as news, videos, tweets, trendingTopics and so on.

The idea is to reduce this payload and the number of data returned. The new payload of `/posts` should be:
```
[
    {
        "id": 1,
        "category": {
            "id": 1,
            "title": "test",
            "creation_date": "2021-02-01T00:06:46Z",
            "description": "tests"
        },
        "trending_topics": [
            {
                "id": 1,
                "title": "#TodosComBolsonaro",
                "creation_date": "2021-02-16T01:57:06Z",
                "trends_position": 5,
            },
            {
                "id": 2,
                "title": "#ForaKarol",
                "creation_date": "2021-02-16T01:57:27Z",
                "trends_position": 4,
            },
            {
                "id": 3,
                "title": "#BBB21",
                "creation_date": "2021-02-16T01:57:51Z",
                "trends_position": 3,
            },
            {
                "id": 4,
                "title": "#VeggieFood",
                "creation_date": "2021-02-16T01:58:06Z",
                "trends_position": 2,
            },
            {
                "id": 5,
                "title": "#BRASIL",
                "creation_date": "2021-02-16T01:58:24Z",
                "trends_position": 1,
            }
        ],
        "title": "2021-02-01",
        "creation_date": "2021-02-01T00:07:10Z",
        "search_count": 0
    }
]
```
and we'll break down the /post into two different requests, the first one is the current `/posts` and the second one will be `/posts/history`. Both will return the payload above, but the first one will retrieve only the current week posts or a empty list if there's no posts registered yet. The other one returns all the loads of a month.
If the `/posts/history` was called without parameters, it will return the list of posts of the current month or a empty list, if there's no posts registered yet. If there's parameters, it should be a year and a month, like: `/posts/history/2020/1` and it will return the list of posts from January 2020 or a empty list if there's no posts registered on that period.

### PR DESCRIPTION
**Pull request by claytonrv**
# Changes Description

Select the type of changes you're doing on project:

- [ ] **Feature**
- [X] **Change**
- [ ] **Fix**
- [ ] **Configuration**

# Solution Descripton

Corrected the project structure and changed the Posts payload to return as expected 

```
[
    {
        "id": 1,
        "category": {
            "id": 1,
            "title": "test",
            "creation_date": "2021-02-01T00:06:46Z",
            "description": "tests"
        },
        "trending_topics": [
            {
                "id": 1,
                "title": "#TodosComBolsonaro",
                "creation_date": "2021-02-16T01:57:06Z",
                "trends_position": 5,
            },
            {
                "id": 2,
                "title": "#ForaKarol",
                "creation_date": "2021-02-16T01:57:27Z",
                "trends_position": 4,
            },
            {
                "id": 3,
                "title": "#BBB21",
                "creation_date": "2021-02-16T01:57:51Z",
                "trends_position": 3,
            },
            {
                "id": 4,
                "title": "#VeggieFood",
                "creation_date": "2021-02-16T01:58:06Z",
                "trends_position": 2,
            },
            {
                "id": 5,
                "title": "#BRASIL",
                "creation_date": "2021-02-16T01:58:24Z",
                "trends_position": 1,
            }
        ],
        "title": "2021-02-01",
        "creation_date": "2021-02-01T00:07:10Z",
        "search_count": 0
    }
]
```

# Tests
**Views:** content/tests/test_views.py
**Serializers:** content/tests/test_serializers.py

# Task

Diana: [Task_#41](https://github.com/Try-tech-Labs/diana/issues/41)

### CODE REVIEW

**Review by mspdl — APPROVED** [review]

File: .env [hunk 1]
```diff
@@ -2,4 +2,4 @@ SQL_ENGINE=django.db.backends.postgresql
 SQL_DATABASE=
 SQL_USER=
 SQL_PASSWORD=
-SQL_HOST=
+SQL_HOST=172.17.0.1
```
Comment (lucasmauro) [inline .env:5]: Nice touch.

File: app/content/models.py [hunk 2]
```diff
@@ -1,12 +1,9 @@
 from django.db import models
-from django.utils import timezone
-from django.contrib.postgres.fields import ArrayField
 
 
 class BaseModel(models.Model):
-    id = models.PositiveIntegerField(primary_key=True)
     title = models.CharField(max_length=280)
-    creation_date = models.DateTimeField(default=timezone.now)
+    creation_date = models.DateTimeField(auto_now_add=True)
```
Comment (lucasmauro) [inline app/content/models.py:6]: Is this the same as `default=timezone.now`?

**Comment by lucasmauro** [comment]
Thanks for the thorough test files and Factories!

File: app/content/models.py (hunk 2 above)
Comment (claytonrv) [reply app/content/models.py:6]: Yep, it's the exact same thing, but in this case doing one (explicit) import less.

**Pull request merged**